pkg/model/components/context.go

```diff
@@ -125,8 +126,16 @@ func Image(component string, clusterSpec *kops.ClusterSpec) (string, error) {
 		return "gcr.io/google_containers/kubedns-amd64:1.3", nil
 	}
 
+
 	if !IsBaseURL(clusterSpec.KubernetesVersion) {
-		return "gcr.io/google_containers/" + component + ":" + "v" + clusterSpec.KubernetesVersion, nil
+		image := "gcr.io/google_containers/" + component + ":" + "v" + clusterSpec.KubernetesVersion
+
+		assetsBuilder := assets.NewAssetBuilder(clusterSpec.Assets)
```

Comment:
Should now be identical to this:

https://github.com/kubernetes/kops/blob/master/pkg/model/components/kubelet.go#L183-L188